buildscripts/make_dependencies.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Build protoc & netty
+set -ev
+
+# Make protoc
+pushd .
+cd /tmp
+git clone https://github.com/google/protobuf.git
```

Comment:
It would probably be faster to use the release tar.gz. That's what we do for the [docker image](https://github.com/grpc/grpc-docker-library/blob/master/grpc_java_base/Dockerfile).